src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -265,6 +267,10 @@ private void StartCollectingColumns()
     private void FinishCollectingColumns()
     {
         _collectingColumns = false;
+        if (_columns.Count == 0)
```

Comment:
It's better to use `Any()`
And why not to use `_manualGrid = !_columns.Any()`

Reply:
Yes, this is better